frontend/libs/studio-components/src/components/StudioTableLocalPagination/StudioTableLocalPagination.mdx

```diff
@@ -18,25 +18,36 @@ import { StudioTableLocalPagination } from './StudioTableLocalPagination';
 <Heading level={2} size='xsmall'>
   Column format
 </Heading>
-<Paragraph>Columns must have both an `accessor` and a `value` property.</Paragraph>
+<ul>
+  <li>The columns prop has two required properties: `accessor` and `heading`.</li>
+  <li>
+    These properties are optional: `sortable`, `headerCellClass`, `bodyCellClass` and
+    `bodyCellFormatter`.
+  </li>
+</ul>
 
 ```tsx
-const columns = [
+export const columns = [
```

Comment:
Why are only the columns exported?

Reply:
Whoops, we don't need the export keyword here. Fixed ✅